Host a Static Website with Hugo and NGINX.md

```diff
@@ -496,29 +495,23 @@ A better approach is to use the pkill command.
 
     pkill hugo
 
-Then show the list of hugo processes.
-
-    ps aux | grep hugo
-
-Which should return output:
+Now `ps aux | grep hugo` should show no hugo servers running:
 
     username 16628  0.0  0.1  14224  1020 pts/1    S+   15:29   0:00 grep --color=auto hugo
 
 ### Web Browser Returns Bad Gateway
 
-Bad Gateway can be returned by the web browser if the Hugo server is not running or is not running at port 1313. 
+"Bad Gateway" can be returned by the web browser if the Hugo server is not running or is not running at port 1313. 
 
 <p align="center">
   <img src="/images/BadGateway.jpg" alt="Bad Gateway web page" /> 
 </p>
 
-One way to find out more about status of the Hugo server is to issue command:
-
-    ps aux | grep hugo
+In this case you can use `ps aux | grep hugo` to verify the server is not running.
 
-If Hugo is stopped the command returns an output similar to:
+Restart the server, then check for any errors:
 
-    username 15873  0.0  0.1  14224  1024 pts/2    S+   12:29   0:00 grep --color=auto hugo
+    hugo server -t YOUR-TEMPLATE-NAME -D --baseUrl="http://blog.YOUR-DOMAIN-NAME" --appendPort=false &
```

Comment:
Hey, while you are testing, verify first that this is the command you want here, but also with the "&", verify that the server keeps running after logging out. I get the feeling it wont.
I think it is better to use the nohup (no hangup command). Here is an example for a nodejs server
`nohup nodejs server.js --be_ip 10.128.0.2 --fe_ip 10.128.0.3 &`
so `nohup server --options ... &`

Reply:
I explored this `&` issue some to see what was the simplest with the least amount of side effect and effort. So, yes the Hugo server keeps running and has been running for several weeks. Even after shutdown and restart and various occasions.

Reply:
There are likely doing something like a hibernate then. NP then as long as it works on Linode.
Btw, I got that nohup nodejs server line from a Google Cloud Kubernetes tutorial for running a nodejs server inside a docker container.

Reply:
For the purposes of this article though, it is probably best to stick with what you have.